haas/cli.py

```diff
@@ -98,16 +99,19 @@ def check_status_code(response):
 
 # TODO: This function's name is no longer very accurate.  As soon as it is
 # safe, we should change it to something more generic.
+
+
```

Comment:
It did.  For all `def`s there has to be two lines.  Obviously we could `# noqa` it but it may be better to just resolve the function's name and take out the todo